cmd/kubeadm/app/util/error.go

```diff
@@ -75,3 +75,11 @@ func checkErr(prefix string, err error, handleErr func(string, int)) {
 		handleErr(err.Error(), DefaultErrorExitCode)
 	}
 }
+
+func FormatErrMsg(errs []error) string {
```

Comment:
this needs a godoc comment `// FormatErrMsg returns a human-readable string describing the slice of errors passed to the function`

Reply:
Please also add an unit test here